test/test_ops.py

```diff
@@ -1000,6 +1002,43 @@ def gen_iou_check(box, expected, tolerance=1e-4):
             gen_iou_check(box_tensor, expected, tolerance=0.002 if dtype == torch.float16 else 1e-3)
 
 
+class TestMasksToBoxes:
+    def test_masks_box(self):
+        def masks_box_check(masks, expected, tolerance=1e-4):
+            out = ops.masks_to_boxes(masks)
+            torch.testing.assert_close(out, expected, rtol=0.0, check_dtype=False, atol=tolerance)
+
+        # Check for int type boxes.
+        def _get_image():
+            assets_directory = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets")
+            mask_path = os.path.join(assets_directory, "masks.tiff")
+            image = Image.open(mask_path)
```

Comment:
maybe keep the context manager for the image to be closed automatically?

Reply:
Hmm, this function returns the image, a with context will close it and make it unavailable for outer function to test?